### ISSUE TITLE
Unable to play music disc more than 1 in subgame version 0.13.0?

### ISSUE THREAD

**Issue by WSDguy2014** [Wuzzy2/MineClone2-Bugs#112]
If first jukebox is playing a music disc, and other jukebox is trying to play another music disc, first jukebox will stop playing music disc and other jukebox plays another music disc without ejecting disc in first jukebox.
Is that glitch?

**Comment by Wuzzy2** [comment]
Yes, what you say is true, it's how I implemented it. The played music is per-player, but maybe it's not how it is supposed to work, I guess.

But how is it supposed to work?
If all music is per-jukebox, I see a problem:
If a player puts in 10 music discs in 10 jukeboxes next to each other, doesn't it get really, REALLY loud and everyone in radius just heas a terrible noise instead of music?

I'm not sure if this is a good idea to implement it like this, even if this would be exactly how Minecraft does it.

**Comment by WSDguy2014** [comment]
Needs new options to change volume sound and music for "Jukebox/Noteblocks", to prevent making loud/terrible noise.
